index.html

```diff
@@ -1,11 +1,87 @@
 <!DOCTYPE html>
 <html lang="pt-BR">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Finbits</title>
-</head>
-<body>
- Em breve 
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="A vida é muito curta para ser burocrática. A Finbits pode reduzir em até 70% o tempo gasto com a gestão das finanças da sua empresa."/>
+    <link rel="icon" href="./images/favicon.png"/>
```

Comment:
```suggestion
    <link rel="icon" href="/images/favicon.png"/>
```

Rola usar url absoluta?